web/src/pages/mgmt/deadurls.tsx

```diff
@@ -15,9 +16,50 @@ const DeadUrlsPage = (props: Props): ReactElement => {
   const [isAuthed, setIsAuthed] = useState<boolean>(false);
   const [password, setPassword] = useState<string>("");
 
+  const generateContent = (): string => {
+    let deadLinkFileOutput = "Possibly Blocked Or Deadlinks \n\n";
+
+    Object.keys(props.deadLinks).map((page) => {
+      if (props.deadLinks[page].length > 0) {
+        deadLinkFileOutput = `${deadLinkFileOutput}\n\n Page: ${page}`;
+        {
+          props.deadLinks[page].map((link) => {
+            deadLinkFileOutput = `${deadLinkFileOutput}\n\t${link}`;
+          });
+        }
+      }
+    });
+    return deadLinkFileOutput;
+  };
+
+  const handleDownloadClick = (): void => {
+    const content = generateContent();
+
+    const blob = new Blob([content], { type: "text/plain" });
+    const blobUrl = URL.createObjectURL(blob);
+
+    const link = document.createElement("a");
+    link.href = blobUrl;
+    link.download = "deadurls.txt";
+    link.textContent = "Download Text File";
+
+    document.body.append(link);
+    link.click();
+
+    URL.revokeObjectURL(blobUrl);
+    link.remove();
+  };
+
   const authedView = (
     <>
       <h1>Dead URLs</h1>
+      <h2>
+        NOTE: this only works locally, (ask a dev to run this locally for results
+        https://www.pivotaltracker.com/story/show/185355762)
+      </h2>
```

Comment:
I wanted to link to the pivotal ticket as the comments contain vital context if we ever want to update this to be available in dev again. 

It feels kinda like a comment in this regard but I would much rather have this context than not.

Reply:
No way to store this message elsewhere in a shared place?  I guess you dont have to if it is used once.